src/App.tsx

```diff
@@ -1,6 +1,12 @@
-import { Loader } from './components/Loader';
+import {
+  NavLink, Navigate, Route, Routes,
+} from 'react-router-dom';
+import cn from 'classnames';
 
 import './App.scss';
+import { HomePage } from './components/HomePage/HomePage';
+import { PeoplePage } from './components/PeoplePage/PeoplePage';
+import { NotFoundPage } from './components/NotFoundPage/NotFoundPage';
```

Comment:
would be better to create `index.ts` file in each of the component folder
`components/ComponentName1/index.ts`
```
export * from './ComponentName1';
```

then, create `index.ts` file in the `components` folder like

`components/index.ts`

```
export * from './ComponentName1';
export * from './ComponentName2';
export * from './ComponentName3';
```

usage

`App.tsx`
```
import { ComponentName1, ComponentName2, ComponentName3 } from './components'
```